relayer-cli/src/commands/light/rm.rs

```diff
@@ -1,35 +1,42 @@
-use std::ops::Deref;
+use std::{io, io::Write, ops::Deref};
 
 use crate::prelude::*;
 
 use abscissa_core::{application::fatal_error, error::BoxError, Command, Options, Runnable};
 
 use ibc::ics24_host::identifier::ChainId;
-use relayer::config::Config;
+use relayer::config::PeersConfig;
 use tendermint_light_client::types::PeerId;
 
 #[derive(Command, Debug, Options)]
 pub struct RmCmd {
-    /// peer id for this client
+    /// identifiers of peers to remove
     #[options(free)]
-    peer_id: Option<PeerId>,
+    peer_id: Vec<PeerId>,
 
+    /// identifier of the chain to remove peers from
     #[options(short = "c")]
-    /// identifier of the chain
     chain_id: Option<ChainId>,
 
     /// force removal of primary peer
     #[options(short = "f")]
     force: bool,
+
+    /// remove all peers, implies --force
+    #[options(no_short)]
+    all: bool,
 }
 
 #[derive(Clone, Debug)]
 struct RmOptions {
-    /// identifier of the chain
+    /// identifier of the chain to remove peers from
     chain_id: ChainId,
 
-    /// peer id for this client
-    peer_id: PeerId,
+    /// identifiers of peers to remove
+    peer_ids: Vec<PeerId>,
+
+    /// remove all peers, implies --force
+    all: bool,
```

Comment:
Can we have a flag to skip confirmation?

Reply:
Right, I forgot about that, will get on it!

Reply:
Done